Shared.IntegrationTests/UploadApi/ArchiveMethodsTest.cs

```diff
@@ -156,35 +154,92 @@ public void TestCreateZip()
         }
 
         [Test]
-        public void TestDownloadArchiveUrl()
+        public void TestDownloadArchiveUrlForImage()
         {
-            var archiveTag = GetMethodTag();
-            var parameters = new ArchiveParams().Tags(new List<string> { archiveTag });
+            var tag = GetMethodTag();
+            var uploadParams = new ImageUploadParams
+            {
+                File = new FileDescription(m_testImagePath),
+                Tags = $"{tag},{m_apiTag}"
+            };
+            var uploadResult = m_cloudinary.Upload(uploadParams);
+            var publicIds = new List<string> { uploadResult.PublicId };
+
+            var archiveParams = new ArchiveParams().PublicIds(publicIds);
+            var archiveUrl = m_cloudinary.DownloadArchiveUrl(archiveParams);
+
+            Assert.True(CanArchiveFileBeDownloaded(archiveUrl));
```

Comment:
If you are already downloading the file, maybe consider checking that it is a valid archive and at least contains amount of files that you expect.